WalletWasabi/Blockchain/Transactions/TransactionHistoryBuilder.cs

```diff
@@ -75,7 +75,7 @@ public List<TransactionSummary> BuildHistorySummary()
 							Label = spenderTransaction.Label,
 							TransactionId = spenderTxId,
 							BlockIndex = spenderTransaction.BlockIndex,
-							IsLikelyCoinJoinOutput = containingTransaction.Transaction.IsLikelyCoinjoin()
```

Comment:
Do we know why we used the `containingTransaction`? Is it leftover or  does it hold logic which i'm not seeing?

Reply:
I am not sure if the change is correct but it makes sense. I think it was a copy/paste error.